delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ResourcesSearchParams.scala

```diff
@@ -62,6 +65,17 @@ final case class ResourcesSearchParams(
 
 object ResourcesSearchParams {
 
+  case class FileUserMetadata(keywords: Map[String, String])
```

Comment:
`name` and `description` are missing.
The key should be a `Label` to restrain the possible values